docs/get-started.md

```diff
@@ -26,13 +26,41 @@ NOTE: Make sure to check out the appropriate version. We recommend that you chec
 
 ### Set up server
 
-1. Start the server and the local storage service. In the root directory of Ark, run:
+These instructions assume that you are running Minio inside your cluster.
+
+1. Start the local storage service. In the root directory of Ark, run:
 
     ```bash
     kubectl apply -f examples/common/00-prereqs.yaml
-    kubectl apply -f examples/minio/
+    kubectl apply -f examples/minio/00-minio-deployment.yaml
     ```
 
+1.  Get the Minio URL:
+
+    - if you're running Minikube
+
+    ```shell
+    minikube service minio --namespace=heptio-ark --url
+    ```
+
+    - in any other environment
+
+       1. Get the value of the IP address or DNS name of any node in your cluster.
+
+       1. Append the value of the NodePort to get a complete URL. You can get this value by running:
+
+       ```shell
+        kubectl -n heptio-ark get svc/minio -o jsonpath='{.spec.ports[0].nodePort}'
+      ```
+
+1. In `examples/minio/05-ark-backupstoragelocation.yaml`, replace NODE_URL_OR_IP:NODE_PORT with the value of the Minio URL.
+
+1. Start the server:
+
+   ```shell
+   kubectl apply -f examples/minio/20-ark-deployment.yaml -f examples/minio/30-restic-daemonset.yaml
```

Comment:
I don't think you need/want the second `-f` there

Reply:
I'll make a point of updating the upstream docs, then ;)

Reply:
looks like I'm wrong here so ignore.  For the record you can either do it like you have it, or put a comma (no space) between file names (e.g. `-f examples/minio/20-ark-deployment.yaml,examples/minio/30-restic-daemonset.yaml`)